WcaOnRails/db/migrate/20190716065618_add_id_to_inbox_results.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddIdToInboxResults < ActiveRecord::Migration[5.2]
+  def change
+    add_column :InboxResults, :id, :primary_key
```

Comment:
note to self: I absolutely need to test this on staging, to see if CAD and posting results are fine with that!

Reply:
I've deployed this to staging and deleted then posted results results for that competition again (with fake personal information of course).
Everything went well (especially on the `InboxResults`->`Results` copy which was the only thing I wondered about, as all the other php scripts work on `Results`).

I'll deploy this to production and inform the WRT.